scripts/ci/build_emscripten.sh

```diff
@@ -34,55 +34,98 @@
 
 set -ev
 
-if test -z "$1"; then
-	BUILD_DIR="emscripten_build"
-else
-	BUILD_DIR="$1"
-fi
+function build() {
+    local build_dir=$1
+    local pre_release=$2
```

Comment:
You could apply the default here instead of checking it with an `if` below.
```suggestion
    local pre_release="${2:-ci}"
```